pysilcam/tests/test_z_classify.py

```diff
@@ -0,0 +1,80 @@
+from pysilcam.silcam_classify import load_model, predict
+from skimage.io import imread
+import glob
+import os
+import numpy as np
+import unittest
+import tensorflow as tf
+
+# Get user-defined path to unittest data folder
+ROOTPATH = os.environ.get('UNITTEST_DATA_PATH', None)
+
+# Get user-defined tensorflow model path from environment variable
+MODEL_PATH = os.environ.get('SILCAM_MODEL_PATH', None)
+
+print('ROOTPATH',ROOTPATH)
+print('MODEL_PATH',MODEL_PATH)
+
+@unittest.skipIf((ROOTPATH is None),
+    "test path not accessible")
```

Comment:
E128: continuation line under-indented for visual indent  <!-- Automatically posted by Lintly -->